etc/schema.json

```diff
@@ -3742,28 +3742,36 @@
             "additionalProperties": false,
             "properties": {
                 "age": {
-                    "type": "integer"
+                    "type": "integer",
+                    "description": "duration of the flow (measured from timestamp of last packet and first packet)"
```

Comment:
nit: we want descriptions to start with a capital letter :)

Reply:
cc @jasonish who created this keywords section in the schema ;-)